contents/docs/integrate/send-events/_snippets/send-events-flutter.mdx

```diff
@@ -8,7 +8,7 @@ import Intro from "./intro.mdx"
 import 'package:posthog_flutter/posthog_flutter.dart';
 
 Posthog.capture(
-  eventName: 'user signed up',
+  eventName: 'user_signed_up',
```

Comment:
Is eventName a built-in one?

Reply:
Im not familiar with flutter, but I think so. It looks like its the parameter name